tests/test_views.py

```diff
@@ -3208,6 +3208,29 @@ def setUpClass(cls):
         cls._original_enable_email_subscriptions = settings.ENABLE_EMAIL_SUBSCRIPTIONS
         settings.ENABLE_EMAIL_SUBSCRIPTIONS = True
 
+    def test_user_cancel_and_choose_help_mailing_list(self):
+        user = AuthUserFactory()
```

Comment:
I'm not sure how that would be accomplished, since the default is for the user to be subscribed to help mail list so we need to first assert that user is no long subscribed to be able to properly test the subscribe call of user choose mailing lists